Library/Homebrew/missing_formula.rb

```diff
@@ -0,0 +1,106 @@
+require "formulary"
+require "tap"
+require "utils"
+
+module Homebrew
+  module MissingFormula
+    class << self
```

Comment:
I can't get `module_function` to work with the `extend` and `alias` here. I think we tried before on `Emoji` and gave up.